python/dockerfile

```diff
@@ -1,4 +1,22 @@
-FROM alpine
+#imageのrepoの名前
+# FROM alpine
+FROM python:3.8
 
-# STEP4-4では以下は変更しない
-CMD ["python", "-V"]
+RUN pip install --upgrade pip
+
+
+WORKDIR /app
+
+RUN pip install -U setuptools pip
+RUN pip install Cython
+RUN pip install wheel
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+COPY . /db
```

Comment:
Make sure this `COPY` method actually works as intended. Is the db directory accessible from this Dockerfile?
Reference: https://riptutorial.com/docker/example/11005/copy-instruction